lib/statsd_middleware.rb

```diff
@@ -44,3 +44,9 @@ def call(env)
     [status, headers, response]
   end
 end
+
```

Comment:
I know it's technically not middleware, but it seems like it belonged with the other request level metrics. I'm open to suggestions on better locations

Reply:
Maybe `config/initializers/statsd.rb` ?